tools/scripts/linux.mk

```diff
@@ -109,8 +109,6 @@ LIB_TINYIIOD_PATH = $(LIBRARIES)/libtinyiiod/build
 LIB_TINYIIOD = tinyiiod
 TINYIIOD_STD_TYPES = _USE_STD_INT_TYPES
 CFLAGS += -D $(TINYIIOD_STD_TYPES)
-EXTRA_LIBS += $(LIB_TINYIIOD)
-EXTRA_LIBS_PATHS += $(LIB_TINYIIOD_PATH)
```

Comment:
If those are removed, what's happening with 
 `
ADD_LIBRARIES = $(foreach lib, $(EXTRA_LIBS), $(call add_library_cmd,$(lib)))
ADD_LIBRARIES_PATH = $(foreach path, $(EXTRA_LIBS_PATHS), $(call lib_serach_path_cmd,$(path)))
`
At the moment, the projects don't use the generic.mk file

Reply:
They are added by create_project.tcl . That's the error was to add the here too, causing the duplication error.